src/testing/Recoil_CreateReactRoot.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails oncall+recoil
+ * @flow strict-local
+ * @format
+ */
+
+const React = require('React');
+const ReactDOM = require('ReactDOM');
+
+function createReactRoot(container: HTMLElement, contents: React.Node) {
+  // To test in Legacy Mode
```

Comment:
i don't think it can be called "Legacy" yet 😅
let's call it "Standard (non-concurrent) mode"

Reply:
Oo, I took this term from the official react doc)